src/campaign_server/campaign_server.cpp

```diff
@@ -893,9 +898,12 @@ void server::handle_upload(const server::request& req)
 		data["icon"] = (*campaign)["icon"];
 		data["type"] = (*campaign)["type"];
 		data["tags"] = (*campaign)["tags"];
-		(*campaign).clear_children("translation");
 		find_translations(data, *campaign);
 
+		for(const config& locale_params : (*campaign).child_range("translation")) {
+			data.add_child("translation", locale_params);//Do we need it?
+		}
+
```

Comment:
To answer your question: no, we don't need it. There's no reason for most of the metadata to be copied into `data` (the on-disk archive's WML), but we seem to do that for historical reasons anyway. Probably best not to waste time with it in this PR.

Reply:
Well, thanks for clarification with this question. I'll remove it then.